### PR TITLE
IBX-6762 Update block reference page

### PR DESCRIPTION
**Pull request by MagdalenaZuba**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | https://issues.ibexa.co/browse/IBX-6762
| Versions      | 4.6
| Edition       | all

#### Checklist

- [ ] Text renders correctly
- [ ] Text has been checked with vale
- [ ] Description metadata is up to date
- [ ] Redirects cover removed/moved pages
- [ ] Added link to this PR in relevant JIRA ticket or code PR

### CODE REVIEW

File: docs/content_management/block_reference.md [hunk 1]
```diff
@@ -205,22 +244,44 @@ On the **Design** tab, in the **View** field, change the layout to "Products" an
...
+On the **Properties** tab, set values in the following fields:
+
+- **Name** – Enter a name for the page block.
+- **Personalization Scenario** – Select "Recently added items" to display products that were recently added to PIM.
+- **Product Types to be displayed** – Select the type of products to be displayed on the list.
+- **Limit** – Set the number of products to be displayed.
+
+## RSS block
+
+Imports and showcase news content from RSS feeds (channels), helping users stay informed with the latest updates and diverse news sources.
```
Comment (julitafalcondusza) [inline docs/content_management/block_reference.md:270]:
```suggestion
Imports and showcases news content from RSS feeds (channels), helping users stay informed with the latest updates and diverse news sources.
```

File: docs/content_management/block_reference.md [hunk 2]
```diff
@@ -36,167 +36,206 @@ The following blocks are provided with a clean installation of [[= product_name
...
+- Refresh — update the block's preview.
+- Delete — remove the existing block.
+
+### Design tab
+
+In the block configuration on the **Design** tab, under the **View** field, you can select the layout for presenting a block.
+
+### Scheduler tab
+
+In the block configuration on the **Scheduler** tab, you can set a date for the block to be revealed and hidden.
```
Comment (julitafalcondusza) [inline docs/content_management/block_reference.md:66]:
```suggestion
In the block configuration on the **Scheduler** tab, you can set a date for the block to be revealed or hidden.
```

**Review by julitafalcondusza — APPROVED** [review]

File: docs/content_management/block_reference.md (hunk 2 above)
Comment (dabrt) [inline docs/content_management/block_reference.md]:
```suggestion
In the block configuration, on the **Design** tab, under the **View** field, you can select the layout for presenting a block.

### Scheduler tab

In the block configuration, on the **Scheduler** tab, you can set a date for the block to be revealed and hidden.
```

File: docs/content_management/block_reference.md [hunk 3]
```diff
@@ -36,167 +36,206 @@ The following blocks are provided with a clean installation of [[= product_name
...
-- **Image** - Click **Select content**, navigate through the content and select an image to display.
-- **URL** - Enter a URL to open when clicking the selected image.
+- **Name** — Enter a name for the page block.
+- **Image** — Click **Select content**, browse the available content, and choose an image for display.
+- **URL** — Enter a URL that opens when users click on the banner block.
 
 ## Bestsellers block
 
-On the **Basic** tab, set values in the following fields:
+Displays products that have recently been top sellers, helping users discover popular items quickly.
```
Comment (dabrt) [inline docs/content_management/block_reference.md:80]:
```suggestion
Displays products that were recently sellers, to help users discover popular items quickly.
```

While it is not wrong, simplified English suggests avoiding perfect tenses and using simple tenses.

File: docs/content_management/block_reference.md [hunk 4]
```diff
@@ -36,167 +36,206 @@ The following blocks are provided with a clean installation of [[= product_name
...
+and published in your PIM.
+On the **Properties** tab, set values in the following fields:
 
-- **Name** – Enter a name for the Page block.
-- **Default catalog** – Select a catalog that is displayed to end-users who are either 
-anonymous or do not belong to any of the customer groups assigned to specific catalogs.
-- **Setup customer group and catalog matching priority rules** - Create at least one assignment: add a row, then select a customer group and a matching catalog. 
-- **Limit** – Set the number of products to be displayed.
+- **Name** – Enter a name for the page block.
+- **Default catalog** – Select a catalog that is displayed to end-users who are anonymous or not part of any customer group assigned to specific catalogs
```
Comment (dabrt) [inline docs/content_management/block_reference.md:99]:
```suggestion
- **Default catalog** – Select a catalog that is displayed to end-users who are anonymous or not part of any customer group assigned to specific catalogs.
```

File: docs/content_management/block_reference.md [hunk 5]
```diff
@@ -36,167 +36,206 @@ The following blocks are provided with a clean installation of [[= product_name
...
+- **Set up customer group and catalog** — Add at least one assignment: add a row, then select a customer group and a matching catalog.
+- **Display limit** – Set the number of products to be displayed.
 
 On the **Design** tab, in the **View** field, select the layout to be used to present a list of products and submit your changes.
 
 ## Code block
 
-On the **Basic** tab, set values in the following fields:
+Enables you to add customized text, links, images, and more to your Page by utilizing custom HTML.
```
Comment (dabrt) [inline docs/content_management/block_reference.md:107]:
```suggestion
Enables you to use custom HTML to add customized text, links, images, and more to your page.
```
Preferential, just like the sound of it better.

File: docs/content_management/block_reference.md [hunk 6]
```diff
@@ -36,167 +36,206 @@ The following blocks are provided with a clean installation of [[= product_name
...
+- **Display limit** — Set the number of products to be displayed.
 
 The rules are checked in order, so when a user belongs to more than one Segment, the first rule applies.
 
 ![Dynamic targeting](img/page_builder_dynamic_targeting.png)
 
 ## Embed block
 
-On the **Basic** tab, set values in the following fields:
+Place any Content item directly on the Page. This function works across all Content Types seamlessly.
```
Comment (dabrt) [inline docs/content_management/block_reference.md:152]:
```suggestion
Place any Content item directly on the page. This function works across all Content Types seamlessly.
```

File: docs/content_management/block_reference.md [hunk 7]
```diff
@@ -36,167 +36,206 @@ The following blocks are provided with a clean installation of [[= product_name
...
 
-- **Name** - Enter a name for the Page block.
-- **Content** - Click **Select content**, navigate through the content and select a Content item. Click **Confirm**.
+- **Name** — Enter a name for the page block.
+- **Content** — Click **Select content**, browse the content, and select a Content item.
 
 ## Form block
 
-Note that completing the settings of the Form block requires at least one Form Content item created.
+Place a selected form from the content structure onto the Page.
```
Comment (dabrt) [inline docs/content_management/block_reference.md:160]:
```suggestion
Place a selected form from the content structure onto the page.
```

File: docs/content_management/block_reference.md [hunk 8]
```diff
@@ -36,167 +36,206 @@ The following blocks are provided with a clean installation of [[= product_name
...
-## Ibexa Connect block
+After submitting the settings, all images in the folder are shown in the Gallery block.
+Selecting a folder containing Content items other than images results in displaying only a link to the folder where they are stored.
 
-On the **Basic** tab, set values in the following fields:
+## Ibexa Connect block
 
-- **Name** – Enter a name for the Page block.
-- **Webhook link** - Enter the link for the Ibexa Connect webhook.
+Access and showcase data retrieved from an Ibexa Connect webhook. Streamline information presentation for seamless insights and improved decision-making
```
Comment (dabrt) [inline docs/content_management/block_reference.md:188]:
```suggestion
Access and showcase data retrieved from an Ibexa Connect webhook. Streamline information presentation for seamless insights and improved decision-making.
```

File: docs/content_management/block_reference.md [hunk 9]
```diff
@@ -36,167 +36,206 @@ The following blocks are provided with a clean installation of [[= product_name
...
 !!! caution "Using Ibexa Connect scenario block"
-    
+
     For more information about using Ibexa Connect scenario block, see [Ibexa Connect scenario block]([[= developer_doc =]]/content_management/pages/ibexa_connect_scenario_block/) in developer documentation.
 
 ## Last purchased block
 
-On the **Basic** tab, set values in the following fields:
+Showcases a list of recently purchased products from PIM.
+Keep track of recent sales and improve product visibility.
```
Comment (dabrt) [inline docs/content_management/block_reference.md:201]:
```suggestion
Helps keep track of recent sales and improve product visibility.
```

File: docs/content_management/block_reference.md [hunk 10]
```diff
@@ -36,167 +36,206 @@ The following blocks are provided with a clean installation of [[= product_name
...
 - **Product Types to be displayed** – Select the type of products to be displayed on the list.
 - **Limit** – Set the number of products to be displayed.
 
-On the **Design** tab, in the **View** field, select the layout to be used to present a list of products and submit your changes.
-
 ## Last viewed block
 
-On the **Basic** tab, set values in the following fields:
+Showcases a list of recently viewed products from PIM.
+Track and show recent product interests for quick access and informed decisions.
```
Comment (dabrt) [inline docs/content_management/block_reference.md:212]:
```suggestion
Helps track and show recent product interests for quick access and informed decisions.
```

File: docs/content_management/block_reference.md [hunk 11]
```diff
@@ -36,167 +36,206 @@ The following blocks are provided with a clean installation of [[= product_name
...
 - **Product Types to be displayed** – Select the type of products to be displayed on the list.
 - **Limit** – Set the number of products to be displayed.
 
-On the **Design** tab, in the **View** field, select the layout to be used to present a list of products and submit your changes.
-
 ## Orders block
 
-On the **Basic** tab, set values in the following fields:
+Showcases a list of orders linked to a specific company or individual customer.
+Track purchase history for streamlined customer service and personalized interactions.
```
Comment (dabrt) [inline docs/content_management/block_reference.md:223]:
```suggestion
Helps track purchase history for streamlined customer service and personalized interactions.
```

File: docs/content_management/block_reference.md [hunk 12]
```diff
@@ -205,22 +244,44 @@ On the **Design** tab, in the **View** field, change the layout to "Products" an
...
+- **Name** – Enter a name for the page block.
+- **URL** — Enter the URL for the RSS news feed.
+- **Limit** — Set the number of news items to be displayed.
+- **Offset** — Set the limit of featured news items to be displayed.
 
 ## Sales representative
 
-On the **Basic** tab, set values in the following fields:
+Imports and showcases sales representatives for the company in the Customer Portal.
+Empowers users with a way to connect with the team handling their sales needs.
```
Comment (dabrt) [inline docs/content_management/block_reference.md:281]:
```suggestion
Gives users a way to connect with the team that handles their sales needs.
```

File: docs/content_management/block_reference.md [hunk 13]
```diff
@@ -291,16 +329,20 @@ You can preview the Page for each of the available Segments:
...
-- **Content** - Enter text, images, videos, etc. using Online Editor.
+- **Name** – Enter a name for the page block.
+- **Content** — Enter text, images, videos, etc. using Online Editor.
 For details, see [Editing Rich Text Fields](create_edit_content_items.md#edit-rich-text-fields).
 
 ## Video block
 
-On the **Basic** tab, set values in the following fields:
+Integrates a video into the Page with standard playback controls.
+Enhance content variety and captivate audiences with seamless video viewing.
```
Comment (dabrt) [inline docs/content_management/block_reference.md:342]:
```suggestion
Helps enhance content variety and captivate audiences with seamless video viewing.
```

**Review by dabrt — APPROVED** [review]

File: docs/content_management/block_reference.md (hunk 3 above)
Comment (MagdalenaZuba) [reply docs/content_management/block_reference.md:80]:
```suggestion
Displays products that were recently top sellers, to help users discover popular items quickly.
```

**Pull request merged**